### PR TITLE
Cu about 20 fix issues relate design

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
SES-FIX-[Trello-20-layout-cu-about](https://trello.com/c/JqoadMnP/20-layout-cu-about)

# Description

# Actions
Fix the text in the back Button y Markdown
Fix Color Toolbar, Align Items , Margin Align Title
Change letter in bottom, correct with avatar , change size icon
Relate mips change color and font-weight
Add space text relate mips, Align title navigation, add Correct margin top contact
Add Padding and color Card-Info-Member,
Modify Icon component to accept margin, add relate margin-bottom
Add Correct padding cu-about

### CODE REVIEW

File: src/stories/components/topbar/topbar.tsx [hunk 1]
```diff
@@ -40,11 +40,11 @@ const AppBar = styled(MuiAppBar, {
 }));
 
 const TopBar = (props: TopBarProps) => {
-  return <AppBar position="absolute" open={props.open}>
+  return <AppBar position="absolute" open={props.open} style={{ backgroundColor: '#C4C4C4' }}>
     <Toolbar
       sx={{
         pr: '24px',
-        backgroundColor: 'white',
+        backgroundColor: '#C4C4C4',
```
Comment (z3phyro) [inline src/stories/components/topbar/topbar.tsx:47]: Let's leave the color as it was if possible. This isn't the final design anyways.

File: src/stories/components/cutable-column-links/cutable-column-links.tsx [hunk 2]
```diff
@@ -31,32 +31,33 @@ interface CutableColumnLinksProps {
...
 };
 
-export const CutableColumnLinks = (props: CutableColumnLinksProps) => {
+export const CutableColumnLinks = ({ width, height, dark, links }: CutableColumnLinksProps) => {
   return <Container>
-    {props.links.map((link, i) => <CustomPopover key={`link-${i}`} title={link.linkType} id={`link-${i}`}>
-      <LinkImage href={link.href} target="_blank">
-        {getImageForLink(link, props.dark)}
+    {links.map((link, i) => <CustomPopover key={`link-${i}`} title={link.linkType} id={`link-${i}`}>
+      <LinkImage href={link.href} target="_blank" width={width ? width + 22 : undefined} height={height}>
```
Comment (z3phyro) [inline src/stories/components/cutable-column-links/cutable-column-links.tsx:59]: What is the reason for this?

File: src/stories/components/button/big-button/big-button.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ font-weight: 500;
 font-size: 12px;
 line-height: 15px;
 text-align: center;
-
+text-transform:none ;
```
[the remark this answers is not in the document]
Comment (z3phyro) [reply src/stories/components/button/big-button/big-button.tsx:27]: Here an example of how they should look. You can find it in cutable-column-summary.tsx
image

[the remark this answers is not in the document]
Comment (z3phyro) [reply src/stories/components/button/big-button/big-button.tsx:27]: Actually you can just pass the object. No need for the arrow function.

File: src/stories/components/cutable-column-links/cutable-column-links.tsx (hunk 2 above)
Comment (z3phyro) [reply src/stories/components/cutable-column-links/cutable-column-links.tsx:59]: Lets create an additional parameter and put it as a right or left margin with sx

File: src/stories/components/cutable-column-links/cutable-column-links.tsx [hunk 4]
```diff
@@ -67,10 +68,14 @@ const Container = styled.div({
   alignItems: 'center'
 });
 
+type StickyLinkProps = {
+  width?: string | number,
+  height?: string | number,
```
Comment (z3phyro) [inline src/stories/components/cutable-column-links/cutable-column-links.tsx:73]: Is there a good reason why we are parsing both strings and numbers here?

**Review by z3phyro — CHANGES_REQUESTED** [review]

File: src/stories/components/button/big-button/big-button.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/components/button/big-button/big-button.tsx:27]: ok, i will write that way

File: src/stories/components/cutable-column-links/cutable-column-links.tsx (hunk 4 above)
Comment (alejandrocabriales) [reply src/stories/components/cutable-column-links/cutable-column-links.tsx:73]: i will let only number

File: src/stories/components/cutable-column-links/cutable-column-links.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/components/cutable-column-links/cutable-column-links.tsx:59]: ok, i will do that, thanks for the recommendation

**Review by z3phyro — CHANGES_REQUESTED** [review]

**Pull request merged**